lib/chef/resource/cron_d.rb

```diff
@@ -97,31 +97,31 @@ def self.validate_dow(spec)
                equal_to: %w{ @reboot @yearly @annually @monthly @weekly @daily @midnight @hourly }
 
       property :minute, [Integer, String],
-               description: "The minute to schedule the cron job to run at. Valid values: 0-59.",
+               description: "The minute at which the cron entry should run (0 - 59).",
                default: "*", callbacks: {
                  "should be a valid minute spec" => ->(spec) { validate_numeric(spec, 0, 59) },
                }
 
       property :hour, [Integer, String],
-               description: "The hour to schedule the cron job to run at. Valid values: 0-23.",
+               description: "The hour at which the cron entry is to run (0 - 23).",
```

Comment:
All the other descriptions use `cron entry should run`

Reply:
Yep. Fixed